### ISSUE TITLE
The string "Templates" seems to be untranslatable

### ISSUE THREAD

**Issue by ryonakano** [elementary/files#542]
![2018-07-11 21 10 14](https://user-images.githubusercontent.com/26003928/42570551-33b40400-854f-11e8-948a-4d3436bd4984.png)

I suppose the string "Templates" in the context menu is not translatable.

On Juno Beta1

**Comment by jeremypw** [comment]
At the moment, the menu shows the actual name of the user's template folder, as given ~/.config/user-dirs.dirs.  I guess that may not be optimal and a translated label could be provided that is always "Templates" (or its translation) wherever the templates are stored.

**Comment by danrabbit** [comment]
I agree that it would probably make sense to just translate this label ourselves. The user probably doesn't care about what the name of the directory templates are stored in is

**Comment by ryonakano** [comment]
Thank you everyone! :smile:
